main.js

```diff
@@ -0,0 +1,27 @@
+const tree = document.getElementById('tree');
+
+function hideElementChildren(event) {
+    const htmlTeg = event.target;
+    if (htmlTeg.nodeName === 'SPAN' && htmlTeg.nextElementSibling !== null) {
+        htmlTeg.nextElementSibling.classList.toggle('hide');
+    }
+};
+
+function textBold(event) {
+    const htmlTeg = event.target;
+    if (htmlTeg.nodeName === 'SPAN') {
+        htmlTeg.classList.add('bold');
+    }
+
+}
+
+function textNorm(event) {

```

Comment:
It is nice you can do it by js. and it will be very useful in the future, but for this case, it is an overkill. `span:hover` in cc will be enaugh